sycl/include/sycl/handler.hpp

```diff
@@ -590,7 +590,9 @@ class __SYCL_EXPORT handler {
 
   /// Saves the location of user's code passed in \p CodeLoc for future usage in
   /// finalize() method.
-  void saveCodeLoc(detail::code_location CodeLoc) { MCodeLoc = CodeLoc; }
+  void saveCodeLoc(detail::code_location CodeLoc);
+  void saveCodeLoc(detail::code_location CodeLoc, bool IsTopCodeLoc);
```

Comment:
```suggestion
  void saveCodeLoc(detail::code_location CodeLoc, bool IsTopCodeLoc = true);
```

Reply:
I obviously did it at first, but eventually, I added a new function just to avoid ABI check failure since on windows, the old saveCodeLoc function is already in the symbols list (it is not on linux).
So, it was made to prevent the below error, still if you prefer to break, just let me know if I need to do anything else to prevent the build failure. Is it acceptable if I remove a line from "sycl_symbols_windows.dump"?

C:\Users\gzadica1\github\llvm>python sycl\tools\abi_check.py --mode check_symbols --reference sycl\test\abi\sycl_symbols_windows.dump build\bin\sycl8.dll
There are missing symbols in the new library. It is a breaking change. Refer to sycl/doc/developer/ABIPolicyGuide.md for further instructions. Do not forget to update ABI version according to the policy.
The following symbols are missing from the new object file:

?saveCodeLoc@handler@_V1@sycl@@AEAAXUcode_location@detail@23@@Z

Reply:
Ah, I see, that makes sense. In that case, please leave a comment to make this change during the next ABI window.

Reply:
Done. Thanks!